openedx/adg/lms/applications/tests/test_helpers.py

```diff
@@ -352,3 +319,18 @@ def test_has_admin_permissions(mocker, is_business_line_admin, is_superuser, is_
     test_user = UserFactory(is_superuser=is_superuser, is_staff=is_staff)
 
     assert has_admin_permissions(test_user) == expected_result
+
+
+@pytest.mark.django_db
+def test_validate_word_limit_text_area():
+    """
+    Check if the `validate_word_limit_text_area` function raises a ValidationError if the total
+    number of words exceed the provided limit
+    """
+    text_area_input = 'test word limit for textarea'
+    assert validate_word_limit_text_area(text_area_input) is None
+
+    text_area_input *= MAX_NUMBER_OF_WORDS_ALLOWED
```

Comment:
Again, why are we performing this heavy operation of multiplication on a string? Shouldn't we be simply testing for the boundry scenarios?

Reply:
Because otherwise I'll have to add a string constant having 200+ words.

Secondly, concat operation isn't that heavy.

Reply:
Got it. My only concern left here is that according to the current implementation, we are testing for 5 words and 1000 words. We should instead be testing for the boundry cases, like 200 words and 201 words. You don't have to create a string constant having 200+ words for that at least. You can do the following for boundary testing:

text_area_input = 'test'

text_area_input* MAX_NUMBER_OF_WORDS_ALLOWED
text_area_input* (MAX_NUMBER_OF_WORDS_ALLOWED+1)

Reply:
@MunamMubashir-arbisoft this concern still remains ^. Any thoughts about this?

Reply:
Sorry, completely missed this comment. I get your point, I'll add more thorough tests for this.